protected/tutorial_actions.php

```diff
@@ -37,17 +37,35 @@
 }
 
 //get request data
-$num = $_REQUEST['num'];
+$num = $_REQUEST['numaccts'];
 if (!is_numeric($num)) {
   process_error("ERROR: number of accounts must be a number");
   exit();
 }
 
 $user_prefix = $_REQUEST['userprefix'];
 $pw_prefix =  $_REQUEST['pwprefix'];
-$org_email =  $_REQUEST['email'];                            
+$org_email =  $_REQUEST['email'];
 $org_phone =  $_REQUEST['phone'];
-$desc = $_REQUEST['desc'];   
+$desc = $_REQUEST['desc'];
+
+// expiration
+if (! array_key_exists('tutexpiration', $_REQUEST)) {
+  process_error("ERROR: Missing expiration (no key)");
+  exit();
+}
+$expire = $_REQUEST['tutexpiration'];
+if (! (isset($expire) && ! is_null($expire) && $expire != "")) {
```

Comment:
`isset ()` does an `is_null()` test (http://php.net/manual/en/function.isset.php), so the `is_null` test is redundant.